paths.py

```diff
@@ -0,0 +1,59 @@
+import os.path
+from os.path import join as pjoin
+
+
+####################################################################################################
+
+class OPPaths:
```

Comment:
@norswap @ayazabbas @eerkaijun I prefer use a separate dir for our whole deployment structure instead coupled in OP monorepo. I think our tools will focus on deployment so that we just need some files generated for deployment, such as binary, genesis file, key file, config file etc. We can generate directly or move to the specific deployment dir, this step is like a provision process. After that, we can reuse this assets to deploy on various different platform which like our design doc.

Reply:
We discussed this on our call: some of these we could move, some of them, there are things we're used in the monorepo (notably `Deploy.sol`) that assume fixed paths, so we can't change them without patching these files too.

I'd say it's nice to have but not a priority. Let's open an issue about it.